core/themes/basis/css/layout.css

```diff
@@ -11,3 +11,30 @@
 .layout .l-messages {
   margin: 0 0 2rem;
 }
+
+/* Push the footer down to the bottom of the page. */
+.layout {
+  box-sizing: border-box;
+  height: 100vh;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-flex-direction: column;
+  -ms-flex-direction: column;
+  flex-direction: column;
+}
+.admin-bar body .layout {
+  padding-top: 33px;
+  margin-top: -33px;
+}
```

Comment:
Are these admin-bar adjustments needed? I tried removing them from the CSS and it didn't seem to have any adverse effect.